README.md

```diff
@@ -1207,9 +1220,9 @@ Query OK, 0 rows affected (0.04 sec)
 
 
 # Run percona-scheduler-admin with --syncusers
-./percona-scheduler-admin --config-file=config.toml --syncusers
+$ percona-scheduler-admin --config-file=config.toml --syncusers
 
-Syncing user accounts from PXC(127.0.0.1:4130) to ProxySQL
+Syncing user accounts from PXC(192.168.56.32) to ProxySQL
```

Comment:
Syncing user accounts from PXC(192.168.54.32:3306) to ProxySQL
Port number is missing. Please fix this

Reply:
Fixed.